bin/sandbox

```diff
@@ -72,9 +72,11 @@ unbundled bundle exec rails generate spree:install \
   --user_class=Spree::User \
   --enforce_available_locales=true \
   --with-authentication=false \
+  --payment-method=none \
   $@
 
 unbundled bundle exec rails generate solidus:auth:install
+unbundled bundle exec rails generate ${extension_name}:install
```

Comment:
Is this supposed to be interpolated?

Reply:
Yes, that's coming from dev-support and it's valid bash syntax, although less common